docs/components/icons/FrontEndLogos.tsx

```diff
@@ -25,7 +25,7 @@ export function FrontEndLogos({ grad, ...props }: IconProps) {
           <path fill="var(--app-bg)" d="M0 0h44.7v44.7H0z" transform="translate(504.3 .6)" />
         </clipPath>
       </defs>
-      <g clip-path="url(#FrontEndLogos-clip1)">
+      <g clipPath="url(#FrontEndLogos-clip1)">
```

Comment:
This was something I had fixed too but not pushed... We got it double fixed now :D